internal/hook/wait_exec_hook_handler.go

```diff
@@ -126,7 +126,7 @@ func (e *DefaultWaitExecHookHandler) HandleHooks(
 		}
 
 		for containerName, hooks := range byContainer {
-			if !isContainerRunning(newPod, containerName) {
+			if !isContainerReady(newPod, containerName) {
 				podLog.Infof("Container %s is not running: post-restore hooks will not yet be executed", containerName)
```

Comment:
also change log message from **running** to **ready**?

Reply:
message updated in 6593ee44105ea9952db2b1a410ba0b9ae7982116